### PR TITLE
fix(console): switch component

### PR DESCRIPTION
**Pull request by charIeszhao**
## Summary

Fix switch component in sign-in-experience. Previously the switch didn't work and couldn't switch on.

## Testing

- [x] Tested in `sign-in-experience` form and the switches work as expected
- [x] Tested in application details page with M2M app type, and the debugger console does not have any React warning.

## Checklist

- [x] This PR is not applicable for the checklist

### CODE REVIEW

File: packages/console/src/pages/ApplicationDetails/components/AdvancedSettings.tsx [hunk 1]
```diff
@@ -66,18 +66,9 @@ function AdvancedSettings({ applicationType, oidcConfig }: Props) {
...
-            render={({ field: { onChange, value } }) => (
-              <Switch
-                label={t('application_details.enable_admin_access_label')}
-                checked={value}
-                onChange={({ currentTarget: { checked } }) => {
-                  onChange(checked);
-                }}
-              />
-            )}
+          <Switch
```
Comment (gao-sun) [inline packages/console/src/pages/ApplicationDetails/components/AdvancedSettings.tsx:69]: why did it worked before?

File: packages/console/src/components/Switch/index.tsx [hunk 2]
```diff
@@ -3,16 +3,16 @@ import { forwardRef } from 'react';
 
 import * as styles from './index.module.scss';
 
-type Props = Omit<HTMLProps<HTMLInputElement>, 'value'> & {
+type Props = HTMLProps<HTMLInputElement> & {
```
Comment (gao-sun) [inline packages/console/src/components/Switch/index.tsx:6]: would you like to add an issue for adding unit tests for fundamental components

File: packages/console/src/pages/ApplicationDetails/components/AdvancedSettings.tsx (hunk 1 above)
Comment (charIeszhao) [reply packages/console/src/pages/ApplicationDetails/components/AdvancedSettings.tsx:69]: It worked, but previously there was a React warning in the debugger console, saying that we are trying to turn an uncontrolled component into a controlled one. So I tried to fix that in #3719, but that fix causes this bug

File: packages/console/src/components/Switch/index.tsx (hunk 2 above)
Comment (charIeszhao) [reply packages/console/src/components/Switch/index.tsx:6]: I've created a project to track this.
https://linear.app/silverhand/project/unit-tests-for-fundamental-components-in-admin-console-79af882de43b

**Review by wangsijie — APPROVED** [review]

File: packages/console/src/pages/ApplicationDetails/components/AdvancedSettings.tsx (hunk 1 above)
Comment (charIeszhao) [reply packages/console/src/pages/ApplicationDetails/components/AdvancedSettings.tsx:69]: Discussed offline and it's good to go

**Pull request merged**